tests/testthat/test-model_performance-various.R

```diff
@@ -13,3 +18,35 @@ if (require("testthat") && require("performance") && require("insight") && requi
     expect_equal(model_performance(m5)$R2, 0.4294224, tolerance = 1e-4)
   })
 }
+
+
+if (require("testthat") && require("performance") && require("DirichletReg")) {
+  set.seed(123)
+  library(DirichletReg)
+
+  ALake <- ArcticLake
+  ALake$Y <- suppressWarnings(DR_data(ALake[, 1:3]))
+
+  # fit a quadratic Dirichlet regression models ("common")
+  res1 <- DirichReg(Y ~ depth + I(depth^2), ALake)
+
+  test_that("model_performance (Dirichlet regression)", {
+    expect_equal(
+      performance::model_performance(res1),
+      structure(
+        list(
+          AIC = -199.993722776335,
+          BIC = -185.021667961168,
+          NA. = NA,
```

Comment:
Why does it return element `NA.` here?

Reply:
As far as I can see, this has to do with the following warning printed during execution:

> 'r2()' does not support models of class 'DirichletRegModel'.

So, instead, the function just introduces `NA`s. Is that correct?
```r
>     performance::r2(res1) 
'r2()' does not support models of class 'DirichletRegModel'.
[1] NA
```

Reply:
Yes, I just saw that, too. I've implemented r2() for DirichletReg (which calls Nagelkerke's R2).